linera-service/tests/fixtures/evm_call_evm_example_counter.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Shouldn't we add the usual copyright note?

Reply:
The Linera Copyright has been added.